src/cf-layout.less

```diff
@@ -1203,16 +1271,28 @@
         });
     }
 
+    &__flush {
+        margin-top: 0 !important;
+        margin-bottom: 0 !important;
+        margin-right: -(@grid_gutter-width / 2);
+        margin-left: -(@grid_gutter-width / 2);
+        .respond-to-min(600px, {
```

Comment:
Fixed in https://github.com/anselmbradford/cf-layout/commit/1e28511ce46482fe4105cc366b5b44edb0bcddb5